dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataset/DataSet.java

```diff
@@ -778,6 +780,16 @@ public void setStyle(ObjectStyle style) {
     this.style = style;
   }
 
+  @JsonProperty
+  @JacksonXmlProperty(namespace = DxfNamespaces.DXF_2_0)
+  public String getDisplayOptions() {
```

Comment:
I think it is correct to type this `String` as the backend has no interest in "understanding" the JSON so there is no use in putting in effort to parse and render the JSON content. However, documentation and developers will benefit from pointing out what this is for OpenAPI. We can do this by annotating it `@OpenApi.Property(ObjectNode.class)` (jackson's `ObjectNode`).

Could you also please add that to the same getter in `Section` in this PR? Thanks!

Reply:
Sure thing